taskcluster-spec/deploy.yml

```diff
@@ -0,0 +1,65 @@
+---
+resources:
+- name: ping-web-deployment
+  apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: ping
+  spec:
+    $let:
+      service: {$eval: "builtService('ping')"}
+    in:
+      selector:
+        matchLabels:
+          app: taskcluster
+          tc-service: ping
```

Comment:
Would the plan be to make this generic as in something like `${service.name}` eventually or make a different resource per service?

Reply:
Yeah, that's the question.  I think I want to keep it a different resource per service, just to avoid having to write some service-generation thing (which wouldn't be very declarative).  That might seem like a great DRY refactor later.. or we might be glad we didn't try. We'll see?